include/zephyr/logging/log_msg2.h

```diff
@@ -141,7 +141,10 @@ enum z_log_msg2_mode {
 #define Z_LOG_MSG2_CBPRINTF_FLAGS(_cstr_cnt) \
 	(CBPRINTF_PACKAGE_FIRST_RO_STR_CNT(_cstr_cnt))
 
+#define Z_LOG_MSG2_CBPRINTF_FLAGS 0
```

Comment:
Since you are removing changes in `log_msg2.h` 2 commits later, maybe squash this?